### PR TITLE
[INS-362] Add gRPC metadata support note

### PR DESCRIPTION
**Pull request by filfreire**
### Summary

Update gRPC section with the fact we now support metadata. This will be valid after https://github.com/Kong/insomnia/pull/4244 is merged and tagged in a release.

### Reason

Created this PR so we update documentation once we merge the gRPC Metadata work done in https://github.com/Kong/insomnia/pull/4244

### Testing

Screenshot 2021-11-30 at 18 06 55

cc @falondarville

### CODE REVIEW

File: docs/insomnia/grpc.md [hunk 1]
```diff
@@ -105,8 +105,9 @@ The 2021.1 release of Insomnia introduces support for environment variables and
 As of version 2021.1, gRPC in Insomnia does not include:
 
 * Support for running gRPC requests in unit tests
-* Support for  or request chaining
-* Support for gRPC metadata or deadlines
+* Support for request chaining
+* ~~Support for gRPC metadata~~ **UPDATE**: as of version CHANGE_THIS_BEFORE_MERGE, gRPC metadata support was introduced [see PR #4244](https://github.com/Kong/insomnia/pull/4244)
```
Comment (develohpanda) [inline docs/insomnia/grpc.md:109]: This will be part of the next major release, so 2022.1!

Comment (develohpanda) [inline docs/insomnia/grpc.md:109]:
```suggestion
* ~~Support for gRPC metadata~~ **UPDATE**: as of version 2021.7, gRPC metadata support was introduced [see PR #4244](https://github.com/Kong/insomnia/pull/4244)
```

**Review by develohpanda — APPROVED** [review]

**Comment by falondarville** [comment]
I don't think we should be crossing items out, just updating the docs. If people want to see revision history, they can look through it. I also don't like that we talk about metadata in the limitations section.

File: docs/insomnia/grpc.md (hunk 1 above)
Comment (falondarville) [inline docs/insomnia/grpc.md:109]:
```suggestion
```

**Comment by filfreire** [comment]
> I don't think we should be crossing items out, just updating the docs. If people want to see revision history, they can look through it. I also don't like that we talk about metadata in the limitations section. Why not just delete the original line?

@falondarville I agree with this, added your suggested edit 👍

**Review by falondarville — APPROVED** [review]
LGTM, thank you!!

**Pull request merged**